pages/_app.tsx

```diff
@@ -61,17 +58,7 @@ function MyApp({ Component, pageProps }: MyAppProps) {
   return getLayout(
     <>
       <Head>
-        <title>{title}</title>
-        <meta name="description" content={description} />
         <link rel="icon" href="/favicon.ico" />
```

Comment:
Should these head items have keys as well incase a tag gets added on a specific page?

Reply:
I added keys for a couple. The more obscure ones don't feel worth it. The keys aren't a perfect solve because a consumer would still have to check for the key values.

Reply:
Thats fair!